client/state/data-layer/wpcom-http/test/actions.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { http } from '../actions';
+
+const version = 'query.apiVersion';
+const namespace = 'query.apiNamespace';
+
+describe( '#http', () => {
+	it( 'should set the apiVersion', () => {
+		const request = http( { apiVersion: 'v1' } );
+
+		expect( request ).to.have.deep.property( version, 'v1' );
```

Comment:
thanks. I saw your comment then forgot about it. I'll try to update here in the tests before merging. it's as good of a PR as any I suppose to fix that.